CHANGELOG.md

```diff
@@ -27,6 +27,7 @@
 
 ## Breaking changes
 
+- The ODES solver was removed ([#3932](https://github.com/pybamm-team/PyBaMM/pull/3932))
```

Comment:
Could you please state the reason and alternatives here (for users who're using this solver).

Reply:
Done